src/javascript/app/Modules/SmartChart/Containers/smart-chart.jsx

```diff
@@ -50,6 +50,7 @@ class Chart extends React.Component {
                 margin={this.props.margin || null}
                 id={this.props.chart_id}
                 isMobile={this.props.is_mobile}
+                enabledNavigationWidget={!!this.props.scroll_to_epoch || !!this.props.start_epoch}
```

Comment:
is the navigation widget only shown in contract mode? think we can use `this.props.is_contract_mode` prop instead

Reply:
@kellybinary it's a good idea. @shayan please use it.
```suggestion
                enabledNavigationWidget={this.props.is_contract_mode}
```